src/list.tsx

```diff
@@ -122,17 +122,19 @@ interface CommonProps {
     extra?: React.ReactNode;
     dataAttributes?: DataAttributes;
     disabled?: boolean;
+    navigable?: boolean;
 }
 
 interface ContentProps extends CommonProps {
     isClickable?: boolean;
     type?: 'chevron' | 'basic' | 'custom' | 'control';
-    right?: React.ReactNode;
+    renderRight?: ({centerY}: {centerY: boolean}) => React.ReactNode;
```

Comment:
this is a breaking change, we should try to avoid it.
A posible solution would be to make `right` polymorphic, allow to receive `React.ReactNode | ({centerY}: {centerY: boolean})  => React.ReactNode`